{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -7,159 +7,63 @@
-
-    """
-    if locations_file and single_location:
-        raise ValueError('You provided both a locations file and a single location to make_specs.')
-
-    if single_location:
-        return [single_location]
-
-    locations_file = Path(locations_file) if locations_file else MODEL_SPEC_DIR / 'locations.txt'
-    with locations_file.open() as f:
-        # Interpret each line that doesn't start with a '#' as a single location.
-        locations = [l for l in f.readlines() if not l.strip().startswith('#') and l.strip()]
-    if not locations:
-        raise ValueError(f'No locations provided in location file {str(locations_file)}.')
-
-    return locations
-
-
-MODEL_SPEC_DIR = (Path(__file__).parent.parent / 'model_specifications').resolve()
 @click.command()
-@click.option('-l', '--locations-file',
-              type=click.Path(dir_okay=False),
-              help=('The file with the location parameters for the template. If no locations file is provided '
-                    'and no single location is provided with the ``-s`` flag, this will default to the locations '
-                    f'file located at {str(MODEL_SPEC_DIR / "locations.txt")}.'))
 @click.option('-t', '--template',
-              default=str(MODEL_SPEC_DIR / 'model_spec.in'),
+              default=str(paths.MODEL_SPEC_DIR / 'model_spec.in'),
               show_default=True,
               type=click.Path(exists=True, dir_okay=False),
               help='The model specification template file.')
-@click.option('-s', '--single-location',
-              default='',
-              help='Specify a single location name.')
+@click.option('-l', '--location',
+              default='all',
+              show_default=True,
+              type=click.Choice(project_globals.LOCATIONS + ['all']),
+              help='Location to make specification for.')
 @click.option('-o', '--output-dir',
-              default=str(MODEL_SPEC_DIR),
+              default=str(paths.MODEL_SPEC_DIR),
               show_default=True,
-              type=click.Path(exists=True, dir_okay=True),
+              type=click.Path(exists=True),
               help='Specify an output directory. Directory must exist.')
-def make_specs(template: str, locations_file: str, single_location: str, output_dir: str) -> None:
+@click.option('-v', 'verbose',
+              count=True,
+              help='Configure logging verbosity.')
+@click.option('--pdb', 'with_debugger',
+              is_flag=True,
+              help='Drop into python debugger if an error occurs.')
+def make_specs(template: str, location: str, output_dir: str, verbose: int, with_debugger: bool) -> None:
     """Generate model specifications based on a template.
 
     The default template lives here:
 
-    ``{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specification/model_spec.in``
+    ``src/{{cookiecutter.package_name}}/model_specification/model_spec.in``
 
```

Comment:
Aren't the locations being populated from `project_globals` rather than a `locations.txt` file?

Reply:
Fixed